app/controllers/candidate_interface/gcse/grade_explanation_controller.rb

```diff
@@ -1,5 +1,23 @@
 module CandidateInterface
   class Gcse::GradeExplanationController < Gcse::BaseController
+    def new
+      set_previous_path
+      @form = CandidateInterface::GcseGradeExplanationForm.build_from_qualification(current_qualification)
+    end
+
+    def create
+      @form = CandidateInterface::GcseGradeExplanationForm.new(update_params)
+
+      if @form.save(current_qualification)
+        redirect_to candidate_interface_gcse_details_edit_year_path(params[:subject])
```

Comment:
Missed this on the first look. I think this messes up the back button on the year form following this form.
```suggestion
        redirect_to candidate_interface_gcse_details_new_year_path(params[:subject])
```